src/factories/calorimetry/CalorimeterHitDigi_factory.h

```diff
@@ -32,13 +32,13 @@ class CalorimeterHitDigi_factory : public JOmniFactory<CalorimeterHitDigi_factor
     ParameterRef<std::vector<std::string>> m_fields {this, "signalSumFields", config().fields};
     ParameterRef<std::string> m_readout {this, "readoutClass", config().readout};
 
-    Service<DD4hep_service> m_geoSvc {this};
+    Service<ServiceSvc_service> m_serviceSvc {this};
```

Comment:
Just want to call you attention to the new service line that would need to be added to factories to use this (and some changes in the algorithm itself).